src/components/organisms/Navbar/NavbarDropdown/__snapshots__/NavbarDropdown.test.tsx.snap

```diff
@@ -122,6 +122,10 @@ exports[`<NavbarDropdown /> should render component 1`] = `
   }
 }
 
+@media (min-width:1280px) {
+
+}
+
```

Comment:
Do we know what is causing these to get added?

Reply:
It's a known issue in https://github.com/styled-components/jest-styled-components/issues/276#issuecomment-574518539